src/main/java/com/guysfromusa/carsgame/entities/MapEntity.java

```diff
@@ -31,10 +33,14 @@
     @Setter
     @Getter
     @Column(nullable = false)
-    private boolean deleted;
+    private String active = ACTIVE;
 
     public MapEntity(String name, String content) {
         this.name = name;
         this.content = content;
     }
+
+    public void deactivate() {
+        active = Long.toString(id);
```

Comment:
why id?

Reply:
constraint is on 2 columns (name, active)
active column has magic value "ACTIVE"
if map is mark as deleted active is set to not "ACTIVE" value, could be anything
but id I am sure not to fail on constraint